js/index.js

```diff
@@ -137,6 +137,94 @@ Metrics.prototype = {
             return str;
         }
     },
+    record_floating_point_event: function(event_category, // For example, 'eng', or 'user'
```

Comment:
the js naming convention is camel case so this should be 'recordFloatingPointEvent'